src/cmd/cli/command/commands.go

```diff
@@ -637,6 +645,11 @@ func printEndpoints(serviceInfos []*v1.ServiceInfo) {
 	}
 }
 
+func isCommandUsingDefangPlayground(cmd *cobra.Command) bool {
+	var defangDefaultCluster = pkg.Getenv("DEFANG_FABRIC", cli.DefaultCluster)
```

Comment:
This is not the right way to detect playground or not: we use the "defang cluster" even in the BYOC case: it's what we talk to for login, delegating the subdomain, and analytics. The right way to detect BYOC is to check the provider, ie. `--provider`.